Tasks/PublishSymbolsV2/task.json

```diff
@@ -237,6 +237,7 @@
     "UnableToIndexSources": "Unable to index sources.",
     "UnexpectedDbghelpdllExpected0Actual1": "Library dbghelp.dll is already loaded from an unexpected path. Expected '{0}'. Actual '{0}'.",
     "UnsupportedSourceProvider0": "Unsupported source provider '{0}' for source indexing.",
-    "Win32Error0FromMethod1": "Encountered Win32 error '{0}' from method '{1}'."
+    "Win32Error0FromMethod1": "Encountered Win32 error '{0}' from method '{1}'.",
+    "Node10NotSupported": "Node v{0} is no longer supported starting 07/31/2024. Please update pipeline agent to use Node 16 or later."
```

Comment:
Couple points:
1. Sounds like the agent version controls which node is running? If yes, we can include in the message which min agent to upgrade, or better yet how about, please upgrade the agent to the latest version.
2. Personally I would not include date and just state "no longer supporting Node10".

Reply:
I have added date because after the given date, node10 task will not work at all. So it gives a time window to customer to upgrade an agent.

Reply:
Need to keep node version in the warning as even though we do not support node10 task, we still need node v10.24.1 to compile task locally and generate files in "_generated" folder

Reply:
That time window is too small for anyone to upgrade the agent. Even if we merge the PR today, it gives the customer just 10 days to upgrade, which isn't reasonable IMO.

Reply:
What is the minimum agent version the customer should upgrade to?